inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/convolution.hpp

```diff
@@ -32,7 +32,8 @@ typedef std::tuple<
         InferenceEngine::Precision,     // Output precision
         InferenceEngine::Layout,        // Input layout
         InferenceEngine::Layout,        // Output layout
-        InferenceEngine::SizeVector,    // Input shapes
+        std::vector<std::vector<size_t>>, // Input shape
```

Comment:
Minor comment, please align the comments

Reply:
Yes, fixed as you commented. Thanks.